webapp/locales/de.json

```diff
@@ -143,27 +148,44 @@
             "successDelete": "Social-Media gelöscht. Profil aktualisiert!"
         },
         "muted-users": {
-            "name": "Stummgeschaltete Benutzer",
-            "explanation": {
-                "intro": "Wenn ein anderer Benutzer von dir stummgeschaltet wurde, dann passiert folgendes:",
-                "your-perspective": "In deiner Beitragsübersicht tauchen keine Beiträge der stummgeschalteten Person mehr auf.",
-                "search": "Die Beiträge von stummgeschalteten Personen verschwinden aus deinen Suchergebnissen."
-            },
-            "columns": {
-                "name": "Name",
-                "slug": "Alias",
-                "unmute": "Entsperren"
-            },
-            "empty": "Bislang hast du niemanden stummgeschaltet.",
-            "how-to": "Du kannst andere Benutzer auf deren Profilseite über das Inhaltsmenü stummschalten.",
-            "mute": "Stumm schalten",
-            "unmute": "Stummschaltung aufheben",
-            "unmuted": "{name} ist nicht mehr stummgeschaltet"
-        },
-        "privacy": {
-            "name": "Privatsphäre",
-            "make-shouts-public": "Teile von mir empfohlene Artikel öffentlich auf meinem Profil",
-            "success-update": "Privatsphäre-Einstellungen gespeichert"
+          "name": "Stummgeschaltete Benutzer",
+          "explanation": {
+            "intro": "Wenn ein anderer Benutzer von dir stummgeschaltet wurde, dann passiert folgendes:",
+            "your-perspective": "In deiner Beitragsübersicht tauchen keine Beiträge der stummgeschalteten Person mehr auf.",
+            "search": "Die Beiträge von stummgeschalteten Personen verschwinden aus deinen Suchergebnissen."
+          },
+          "columns": {
+            "name": "Name",
+            "slug": "Alias",
+            "unmute": "Entsperren"
+          },
+          "empty": "Bislang hast du niemanden stummgeschaltet.",
+          "how-to": "Du kannst andere Benutzer auf deren Profilseite über das Inhaltsmenü stummschalten.",
+          "mute": "Stumm schalten",
+          "unmute": "Stummschaltung aufheben",
+          "unmuted": "{name} ist nicht mehr stummgeschaltet"
+        },
+        "blocked-users": {
+          "name": "Blocked users",
+          "explanation": {
+            "intro": "Wenn ein anderer Benutzer von dir blockiert wurde, dann passiert folgendes:",
+            "your-perspective": "Du kannst keine Beiträge der blockierten Person mehr kommentieren.",
+            "their-perspective": "Die blockierte Person kann deine Beiträge nicht mehr kommentieren",
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-01-23T17:18:14Z" title="Thursday, January 23rd 2020, 6:18:14 pm +01:00">Jan 23, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -143,27 +148,43 @@
             "successDelete": "Social-Media gelöscht. Profil aktualisiert!"
         },
         "muted-users": {
-            "name": "Stummgeschaltete Benutzer",
-            "explanation": {
-                "intro": "Wenn ein anderer Benutzer von dir stummgeschaltet wurde, dann passiert folgendes:",
-                "your-perspective": "In deiner Beitragsübersicht tauchen keine Beiträge der stummgeschalteten Person mehr auf.",
-                "search": "Die Beiträge von stummgeschalteten Personen verschwinden aus deinen Suchergebnissen."
-            },
-            "columns": {
-                "name": "Name",
-                "slug": "Alias",
-                "unmute": "Entsperren"
-            },
-            "empty": "Bislang hast du niemanden stummgeschaltet.",
-            "how-to": "Du kannst andere Benutzer auf deren Profilseite über das Inhaltsmenü stummschalten.",
-            "mute": "Stumm schalten",
-            "unmute": "Stummschaltung aufheben",
-            "unmuted": "{name} ist nicht mehr stummgeschaltet"
-        },
-        "privacy": {
-            "name": "Privatsphäre",
-            "make-shouts-public": "Teile von mir empfohlene Artikel öffentlich auf meinem Profil",
-            "success-update": "Privatsphäre-Einstellungen gespeichert"
+          "name": "Stummgeschaltete Benutzer",
+          "explanation": {
+            "intro": "Wenn ein anderer Benutzer von dir stummgeschaltet wurde, dann passiert folgendes:",
+            "your-perspective": "In deiner Beitragsübersicht tauchen keine Beiträge der stummgeschalteten Person mehr auf.",
+            "search": "Die Beiträge von stummgeschalteten Personen verschwinden aus deinen Suchergebnissen."
+          },
+          "columns": {
+            "name": "Name",
+            "slug": "Alias",
+            "unmute": "Entsperren"
+          },
+          "empty": "Bislang hast du niemanden stummgeschaltet.",
+          "how-to": "Du kannst andere Benutzer auf deren Profilseite über das Inhaltsmenü stummschalten.",
+          "mute": "Stumm schalten",
+          "unmute": "Stummschaltung aufheben",
+          "unmuted": "{name} ist nicht mehr stummgeschaltet"
+        },
+        "blocked-users": {
+          "name": "Blocked users",
+          "explanation": {
+            "intro": "Wenn ein anderer Benutzer von dir blockiert wurde, dann passiert folgendes:",
+            "your-perspective": "In deiner Beitragsübersicht tauchen keine Beiträge der blockierten Person mehr auf.",
+            "their-perspective": "Umgekehrt das gleiche: Die blockierte Person sieht deine Beiträge auch nicht mehr in ihrer Übersicht.",
```


```suggestion
            "their-perspective": "Die blockierte Person kann deine Beiträge nicht mehr kommentieren",
```
Translates to "The blocked person cannot comment on your posts anymore".